src/main/java/AutoLotto/model/LottoWinChecker.java

```diff
@@ -1,59 +1,60 @@
 package autoLotto.model;
 
-import java.util.HashMap;
+import java.util.EnumMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
 public class LottoWinChecker {
-    private Map<PrizeEnum, Integer> winLottos;
+    private Lotto winLotto;
```

Comment:
보너스 번호도 당첨 번호의 일부인데요
보너스 번호를 당첨 비교 시 주입받기 보다 생성자에서 주입받아 필드로 가지면 어떨까요?
추가로 재할당을 막기 위해 final 키워드를 추가하면 좋을 것 같아요 
```suggestion
    private final Lotto winLotto;
    private final LottoNumber bonusNumber;
```